routemaster/client.rb

```diff
@@ -11,149 +14,171 @@
 
 module Routemaster
   class Client
+    class << self
```

Comment:
`Routemaster::Client` probably shouldn't be a class anymore (as instances of it are useless).